python/lsst/sdm/schemas/cdb_latiss.yaml

```
@@ -1128,6 +1161,10 @@ tables:
     ivoa:ucd: pos.outline;obs.field
 - name: ccdvisit1_quicklook
   "@id": "#ccdvisit1_quicklook"
+ c description: >
```

```suggestion
   description: >
```

This typo is crashing all the PR checks.